packages/story-editor/src/utils/storyUpdates.ts

```diff
@@ -22,20 +22,35 @@
  */
 import cleanForSlug from './cleanForSlug';
 
+interface updateStoryProps {
+  properties: {
+    slug: string;
+  };
+}
+
+interface updateSlugProps {
+  currentSlug: string;
+  currentTitle: string;
+  updateStory: (props: updateStoryProps) => void;
+}
+
 /**
  * For use on blur of updating story title
  * that can impact what story slug is if
  * not set independently.
  *
- * @param {Object} args Necessary data for update.
- * @param {string} args.currentSlug The currently assigned value to story's slug.
- * @param {string} args.currentTitle The currently assigned value to story's title.
- * @param {Function} args.updateStory The callback to useStory's action for updating the story with new slug.
- * @return {void}
+ * @param args Necessary data for update.
+ * @param args.currentSlug The currently assigned value to story's slug.
+ * @param args.currentTitle The currently assigned value to story's title.
+ * @param args.updateStory The callback to useStory's action for updating the story with new slug.ß
  */
-export const updateSlug = ({ currentSlug, currentTitle, updateStory }) => {
+export const updateSlug = ({
+  currentSlug,
+  currentTitle,
+  updateStory,
+}: updateSlugProps) => {
   if (!currentSlug) {
-    const cleanSlug = encodeURIComponent(cleanForSlug(currentTitle));
+    const cleanSlug = encodeURIComponent(String(cleanForSlug(currentTitle)));
```

Comment:
Shouldn't `cleanForSlug` have `string` as a return type already?

Reply:
Regardless, `cleanForSlug` should have `string` as a return type, no? Maybe `cleanForSlug` needs an update then, too.

Reply:
Fixed in https://github.com/GoogleForCreators/web-stories-wp/pull/12737/commits/0648200ccafe4d71057a9c37a2355e9ba7e3ebb1

Reply:
Fixed in https://github.com/GoogleForCreators/web-stories-wp/pull/12737/commits/0648200ccafe4d71057a9c37a2355e9ba7e3ebb1